data/chainData.js

```diff
@@ -337,7 +357,8 @@ export const prefixToId = {
     "bcna": 12,
     "gravity": 13,
     "stride": 14,
-    "pylo": 15
+    "pylo": 15,
+    "whale": 16,
```

Comment:
i think it should be migaloo

Reply:
The token ticker is whale, chain name, and address prefix is migaloo

Reply:
oh this is for mapping prefix to array id

Reply:
so migaloo would do